core/modules/image/image.field.inc

```diff
@@ -125,12 +125,12 @@ function image_field_instance_settings_form($field, $instance) {
   $min_dimensions[1] = empty($min_dimensions[1]) ? NULL : $min_dimensions[1];
   $form['min_dimensions'] = array(
     '#type' => 'item',
-    '#title' => t('Minimum image dimensions'),
+    '#title' => t('Minimum image dimensions (width x height)'),
```

Comment:
```suggestion
    '#title' => t('Maximum image dimensions (width × height)'),
```
Suggest using Unicode "multiplication" rather than letter x.

Reply:
This change recommends `Maximum` over `Minimum` (I think that was a typo)